### PR TITLE
Incremental-alter-configs-implementation(KIP-339)

### PR DESCRIPTION
**Pull request by PrasanthV454**

### CODE REVIEW

File: src/Confluent.Kafka/Impl/SafeKafkaHandle.cs [hunk 1]
```diff
@@ -1517,6 +1517,59 @@ private void setOption_completionSource(IntPtr optionsPtr, IntPtr completionSour
...
+                    var error = Librdkafka.ConfigResource_add_incremental_config(resourcePtr, rc.Name, rc.IncrementalOperation, rc.Value);
+                    if (error != IntPtr.Zero)
+                    {
+                        throw new KafkaException(new Error(error, true));
+                    }
+                }
+                configPtrs[configPtrsIdx++] = resourcePtr;
+            }
+
+            options = options == null ? new IncrementalAlterConfigsOptions() : options;
```
Comment (anchitj) [inline src/Confluent.Kafka/Impl/SafeKafkaHandle.cs:1557]: [Nit] Can be simplified to `options = options ?? new IncrementalAlterConfigsOptions();`

**Review by anchitj — APPROVED** [review]
LGTM. Nitpick comments. Verified locally all the integration tests are passing.

**Review by anchitj — APPROVED** [review]

**Comment by anchitj** [comment]
@emasab Thanks for the changes. Should we merge these changes now or wait until we include the librdkafka.redist with KIP-339? My only concern is that integration tests may be blocked until then.

**Comment by kmcclellan** [comment]
Hi. Concerning https://github.com/kmcclellan/kafka-di/issues/12, please consider creating new interfaces to introduce new API members (or bumping the major version).

**Pull request merged**